### DISCUSSION TITLE
Are there any way to batch search the 2 mgf files between query mgf and database mgf?

### DISCUSSION THREAD

**Discussion by YUANMENG-1** [YuanyueLi/SpectralEntropy#14] [General]

**Comment by YuanyueLi** [comment]
Hi,

Yes, you can try this one: https://github.com/YuanyueLi/EntropySearch

This contains a GUI to let you search one file against another, you can try with the .mgf file format, if the software can't work, send me the .mgf file, I can figure it out.


Best,
Yuanyue

**Comment by YuanyueLi** [comment]
Hi,

We have updated the Entropy Search software (https://github.com/YuanyueLi/EntropySearch).

Now all the major platforms (Windows, Linux, and MacOS) are supported, and many files like .mzML / .mgf / .msp are supported.


Best,
Yuanyue
